.eslintrc

```diff
@@ -2,6 +2,9 @@
   "parser": "babel-eslint",
   "extends": ["airbnb", "prettier"],
   "plugins": ["react", "jsx-a11y", "import", "prettier"],
+  "env": {
```

Comment:
Since we are using jest to run our tests, we could use the jest env instead so:
```json
"env": {
  "jest": true
}
```
That means you could also get rid of the `"jest"` global.

Reply:
Fixed in this pull request + added new pull request for .idea.